apps/voting/contracts/Voting.sol

```diff
@@ -63,6 +65,7 @@ contract Voting is IForwarder, AragonApp {
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
 
+    bool public earlyExecutionDisallowed;
```

Comment:
We could make it a private state variable and implement a `earlyExecutionAllowed` getter, as the rest of the contract's external interface (including the `ChangeEarlyExecution` event) uses `allowed` instead of `disallowed`